src/main/java/io/percy/appium/providers/AppAutomate.java

```diff
@@ -162,4 +163,28 @@ public String deviceName(String deviceName, JSONObject result) {
         return result != null ? result.getString("deviceName") : null;
     }
 
+    public Boolean verifyCorrectAppiumVersion() {
+        Map bstackOptions = (Map) driver.getCapabilities().getCapability("bstack:options");
+        Object appiumVersionJsonProtocol = driver.getCapabilities().getCapability("browserstack.appium_version");
+        if (bstackOptions == null && appiumVersionJsonProtocol == null) {
+            AppPercy.log("Unable to fetch Appium version, "
+                    + "Appium version should be >= 1.19 for Fullpage Screenshot", "debug");
+        } else if ((appiumVersionJsonProtocol != null && !appiumVersionCheck(appiumVersionJsonProtocol.toString()))
+                || (bstackOptions != null && !appiumVersionCheck(bstackOptions.get("appiumVersion").toString()))) {
+            AppPercy.log("Appium version should be >= 1.19 for Fullpage Screenshot, "
```

Comment:
Add these as a warn message. I have implemented it in c# sdk that changes color to yellow. https://github.com/percy/percy-appium-dotnet/blob/main/Percy/AppPercy.cs#L87

Reply:
added warn msgs